### PR TITLE
🐛 Fix #767

### LINKED ISSUE #767: [BUG] Unable to Return to Stream Selection from Clipping Page
## Description
After selecting an event and stream on the `/studio/clips` page and proceeding to the clipping interface, users are unable to navigate back to the stream selection page. This lack of a return option limits user control and navigation flexibility within the clipping feature.

## Steps to Reproduce
1. Navigate to `/studio/clips`
2. Select an event and stream
3. Attempt to return to the stream selection page

## Expected Behavior
A clearly visible button or navigation option should be present, allowing users to return to the stream selection page from the clipping interface.

## Screenshot
Clipping Interface Without Return Option

### PR DESCRIPTION
**Pull request by Mario-SO**
# Pull Request Info

## Description

Used our custom useSearchParams hook and useRouter from Next to handle the go back and also made it to only show the button when necessary

Fixes #767 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

## Screenshots (if appropriate):

https://github.com/user-attachments/assets/6fb09610-d393-4b11-8bad-9b5fcad4eead

## Additional context:

Not really a bug but anyway...

closes #767

### CODE REVIEW

**Comment by greatsamist** [comment]
image
Go back button is missing on mobile nav header

**Comment by greatsamist** [comment]
Looks good, but can we try adding the back button to the clip page instead, like we have on other studio pages
image

**Comment by xannyxs** [comment]
Screenshot 2024-08-12 at 15 43 15

It seems that the icon is not aligned with the text. I also agree with Samuels earlier commment

**Comment by xannyxs** [comment]
> image Go back button is missing on mobile nav header

We do not support Mobile on `studio`. Never hurts to do it, but it is atm not necessary.

File: packages/app/components/Layout/HomePageNavbar.tsx [hunk 1]
```diff
@@ -183,11 +196,24 @@ const PCNavBar = ({
...
-            </Button>
-          </Link>
+          <div className="flex flex-row space-x-1">
+            {showGoBack && (
+              <Button
+                className="hidden lg:block"
+                variant={'outline'}
+                onClick={handleGoBack}
+              >
+                <div className="flex">
```
Comment (greatsamist) [inline packages/app/components/Layout/HomePageNavbar.tsx:228]:
```
<div className="flex items-center">
```

**Review by greatsamist — APPROVED** [review]

**Pull request merged**